cli/ping.go

```diff
@@ -58,6 +58,9 @@ func (r *RootCmd) ping() *serpent.Command {
 				_, _ = fmt.Fprintln(inv.Stderr, "Direct connections disabled.")
 				opts.BlockEndpoints = true
 			}
+			if r.noNetworkTelemetry {
+				opts.EnableTelemetry = true
+			}
```

Comment:
Something seems off with the naming here.

Reply:
I left a comment with the `EnableTelemetry` definition on this, the reasoning being that we default initialize `DialAgentOptions` in tests, where it makes the most sense to have network telemetry disabled.

Reply:
I'm just saying the naming doesn't make a lot of sense. If there is no network telemetry, why would we enable telemetry?

Reply:
oops